lua/ulx/modules/sh/yeet.lua

```diff
@@ -0,0 +1,38 @@
+local CATEGORY_NAME = "Fun"
+
+local MIN_STRENGTH = 6000	
+local MAX_STRENGTH = 500000	
+local DEFAULT_STRENGTH = 6000 
+
+function yeetPlayer( caller, targets, strength )
+	
+	for _, v in pairs( targets ) do 
```

Comment:
Not a big deal, but it's generally bad practice to use single-letter variables. If you are going to use them, you should try to make them "make sense" in the context.

In this case, I think `ply` is the best choice, but if you were going to use a single character, `p` might be more appropriate.